members/templates/members/partials/lendings_card.html

```diff
@@ -1,4 +1,4 @@
-<div class="card col-sm-12 col-xl-8" style="width: 18rem;">
+<div class="card col-sm-12 col-xl-7" style="width: 18rem;">
```

Comment:
![afbeelding](https://github.com/user-attachments/assets/412cc2d6-a52e-478d-a412-a38dbc921b34)
something not going right if i put my browser on mobile

Reply:
I think this is already broken on prod?

Reply:
It was; now it's fixed tho.